cat.html

```diff
@@ -36,26 +41,24 @@ <h2>Cat Form</h2>
         <form action="https://freecatphotoapp.com/submit-cat-photo">
           <fieldset>
             <legend>Is your cat an indoor or outdoor cat?</legend>
-            <label><input id="indoor" type="radio" name="indoor-outdoor" value="indoor" value="loving" > Indoor</label>
-
-            <label><input id="outdoor" type="radio" name="indoor-outdoor" value="outdoor" value="loving"> Outdoor</label>
+            <label><input id="indoor" type="radio" name="indoor-outdoor" value="indoor" checked> Indoor</label>
+            <label><input id="outdoor" type="radio" name="indoor-outdoor" value="outdoor"> Outdoor</label>
           </fieldset>
-
-          
           <fieldset>
             <legend>What's your cat's personality?</legend>
-
-            <input id="loving" type="checkbox" name="personality"> <label for="loving" value = "loving" >Loving</label>
-
-            <input id="lazy" type="checkbox" name="personality"       value = "loving" > <label for="lazy">Lazy</label>
-
-            <input id="energetic" type="checkbox" name="personality"   value = "loving" > <label for="energetic"> Energetic</label>
+            <input id="loving" type="checkbox" name="personality" value="loving" checked> <label for="loving">Loving</label>
+            <input id="lazy" type="checkbox" name="personality" value="lazy"> <label for="lazy">Lazy</label>
+            <input id="energetic" type="checkbox" name="personality" value="energetic"> <label for="energetic">Energetic</label>
           </fieldset>
           <input type="text" name="catphotourl" placeholder="cat photo URL" required>
           <button type="submit">Submit</button>
-          
         </form>
       </section>
     </main>
+    <footer>
+      <p>
+        No Copyright - <a href="https://www.freecodecamp.org">freeCodeCamp.org</a>
+      </p>
+    </footer>
   </body>
 </html>
```

Comment:
Add a new line at the end of the file. See more info [here](https://gist.github.com/OleksiyRudenko/d51388345ea55767b7672307fe35adf3).